qingstor/sdk/config.py

```diff
@@ -87,7 +89,14 @@ def install_default_user_config(self):
 
     def load_config_from_data(self, data):
         for (k, v) in data.items():
+            value = self.load_from_env(k)
+            if not value is None:
```

Comment:
```suggestion
            if value is not None:
```

Reply:
BTW, what will happen if user's config do not have key `endpoint`?

Reply:
if user's config have endpoint, we use endpoint to replace protocol, host and port. if user's config do not have key endpoint,We use protocol, host and port config directly

Reply:
But `data` here is read from user's config, if user's config don't have `endpoint`, we will not read this value from env.

Think about the case like following:

user has config:

```yaml
access_key_id: abc
secret_access_key: def
host: qingstor.com
```

The user start program with env `endpoint: abc.com`, current implemation could ignore `endpoint` and use `host` directly.